modules/admin-ui-frontend/app/scripts/shared/services/jsHelperService.js

```diff
@@ -398,6 +398,27 @@ angular.module('adminNg.services')
             }
           });
         });
+      },
+
+      /**
+       * Removes illegal XML characters from the input string
```

Comment:
The list below suggests that these are the illegal characters while in the regular expression below, these are actually the safe characters.

```suggestion
       * Removes illegal XML characters from the input string.
       * Illegal characters are all characters not in the following list:
```